### PR TITLE
chore: update upgrade e2e documentation

### PR DESCRIPTION
**Pull request by latin-panda**
# Description
Updates upgrade e2e documentation

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
Hi @dianabarsan, can you please have a look? It's 1 line

File: content/en/contribute/code/core/automated-tests.md [hunk 1]
```diff
@@ -201,7 +201,8 @@ To run the upgrade e2e tests in your local environment, follow these steps:
     - `export MARKET_URL_READ=https://staging.dev.medicmobile.org`.
     - `export STAGING_SERVER=_couch/builds_4`.
     - `export BRANCH=<your branch name>`.
-    - `export BASE_VERSION=<base version>`(it can be used `latest` as the value).
+    - `export BASE_VERSION=<CHT base version>`(it can be used `latest` as the value).
+    - `export TAG=<CHT version>`(e.g. `4.8.1`).
```
Comment (dianabarsan) [inline content/en/contribute/code/core/automated-tests.md:205]: This makes it look like the TAG is required, but it's not.

Comment (latin-panda) [reply content/en/contribute/code/core/automated-tests.md]: I've added `Optional`

Comment (latin-panda) [reply content/en/contribute/code/core/automated-tests.md]: @dianabarsan can I merge?

**Review by dianabarsan — APPROVED** [review]
👍🏻

**Pull request merged**